script.js

```diff
@@ -1,6 +1,18 @@
 function getContacts(str) {
-  // your code here
-}
+
+var splitsatu = str.split(',')
+
+var splitdua =[]
+for (i = 0 ; i < splitsatu.length ; i++){
+ splitdua[i]= splitsatu[i].toString().split(':')
```

Comment:
```suggestion
 splitdua[i]= splitsatu[i].split(':')
```
gak perlu pakai `.toString` karena nilainya udah string